tests/expectations/metrics/test_metrics_util.py

```diff
@@ -272,3 +274,149 @@ def test_get_unexpected_indices_for_multiple_pandas_named_indices_named_unexpect
         "Error: The list of domain columns is currently empty. Please check your "
         "configuration."
     )
+
+
+def test_get_dbms_compatible_metric_domain_column_kwargs_column_not_found():
```

Comment:
These tests should have markers

Reply:
@anthonyburdi Good catch -- markers added (committed).  Thanks.